css/cart.css

```diff
@@ -0,0 +1,154 @@
+@import url("/style.css");
+@import url("/css/wishlist.css");
+
+.cart {
+  display: flex;
+  width: 50%;
+}
+
+.cart .img-container {
+  width: 50%;
+}
+
+.cart .img-container img {
+  width: 100%;
+  min-height: 100%;
+  border-radius: 1rem;
+  padding: 1rem;
+}
+
+.cart-info {
+  width: 50%;
+  padding: 1.2rem;
+  background-color: #ffffff;
+}
+
+.title {
+  font-size: 3rem;
+  font-weight: 600;
+}
+
+.price {
+  padding: 1rem 0;
+  letter-spacing: 0.6rem;
+}
+
+.price #nonstrike {
+  font-size: 2rem;
+  padding-right: 1rem;
+}
+
+.price #strike {
+  text-decoration: line-through;
+  font-size: 1.6rem;
+  color: #9a9a9a;
+}
+
+.discount {
+  color: #9a9a9a;
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+.quantity {
+  font-size: 2rem;
+}
+
+#quantityno {
+  border: 0.1rem black solid;
+  border-radius: 0.2rem;
+  width: 20%;
+  padding-left: 1rem;
+}
+
+.btn .remove {
+  width: 8rem;
+  height: 2.5rem;
+}
+
+.btn .move {
+  width: 8rem;
+  height: 2.5rem;
+}
+
+.product-card {
+  width: 30%;
+  min-height: 100%;
+  padding: 1.2rem;
+  font-size: 1.6rem;
+  letter-spacing: 0.3rem;
+}
+
+.product-card .info {
+  text-align: left;
+}
+
+.product-card .title {
+  font-size: 2rem;
+  font-weight: 700;
+  border-bottom: 1px solid black;
+  padding-bottom: 1.2rem;
+  margin-bottom: 1.2rem;
+}
+
+.product-card .total {
+  font-weight: 700;
+  border-bottom: 1px solid black;
+  border-top: 1px solid black;
+  padding: 10px 0;
+  margin-top: 1.2rem;
+}
+
+.price-details #price-detail {
+  float: right;
```

Comment:
suggestion: using float is not  good practice,use flex properties for "justify-content:flex-end or flex-start"